app/src/main/java/org/oppia/android/app/utility/datetime/DateTimeUtil.kt

```diff
@@ -30,4 +31,49 @@ class DateTimeUtil @Inject constructor(
     /** Returns [DateTimeUtil] for the current Dagger graph. */
     fun getDateTimeUtil(): DateTimeUtil
   }
+
+  /**
+   * Returns the readable string of the duration from the provided time in [Long].
+   */
+  fun timeAgoFromTimestamp(timestamp: Long, referenceTime: Long): String {
```

Comment:
Please add new tests for this method to cover all possible pluralizations that should now be supported.